dspy/adapters/types/tool.py

```diff
@@ -204,6 +204,21 @@ def from_langchain(cls, tool: "BaseTool") -> "Tool":
 
         Returns:
             A Tool object.
+
+        Example:
+        ```python
+        from langchain.tools import tool
+        import dspy
+
+        @tool
+        def add(x: int, y: int):
+            "Add two numbers together."
+            return x + y
+
+        tool = dspy.Tool.from_langchain(add)
+        print(await tool.acall(x=1, y=2))
```

Comment:
Simply copying this code to local editor won't work, this code needs to be wrapped by `asyncio.run()`.